### PR TITLE
feat: added table component

### PR DESCRIPTION
**Pull request by assasin0076**

### CODE REVIEW

**Comment by zoltanszogyenyi** [comment]
Hey @assasin0076,

Thanks for making the PR! @cogor can you please check it out?

**Comment by assasin0076** [comment]
Hi, @zoltanszogyenyi . Its not ready yet) I just started my work

**Comment by cogor** [comment]
LGTM

File: src/components/Table/Table.vue [hunk 1]
```diff
@@ -1,104 +1,29 @@
 <template>
-  <div class="relative overflow-x-auto shadow-md sm:rounded-lg">
+  <div class="vp-raw relative overflow-x-auto shadow-md sm:rounded-lg">
```
Comment (cogor) [inline src/components/Table/Table.vue:2]:
```suggestion
  <div class="relative overflow-x-auto shadow-md sm:rounded-lg">
```

**Pull request merged**